drupal8-example/composer.json

```diff
@@ -20,14 +20,14 @@
         "cweagans/composer-patches": "^1.6",
         "drupal-composer/drupal-scaffold": "^2.2",
         "drupal/console": "^1.0.2",
-        "drupal/core": "~8.5.1",
-        "drush/drush": "^8.1",
+        "drupal/core": "^8.6",
+        "drush/drush": "^9",
         "vlucas/phpdotenv": "^2.4",
+        "webflo/drupal-core-require-dev": "^8.6",
```

Comment:
Do you by any chance know why this goes from the  require-dev section to require?

Reply:
Probably because I did `composer require drupal/core:^8.6 webflo/drupal-core-require-dev:^8.6` to get it from `^8.5` to `^8.6`.

Should maybe have done that in a 2-step process with a  `--dev` on the latter?

Reply:
I moved it back.